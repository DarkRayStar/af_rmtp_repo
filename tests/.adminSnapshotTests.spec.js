import React, {} from 'react';
import AdminHome  from '../src/components/admin-components/admin-home.component';
const renderer = require('react-test-renderer');


describe("Snapshot testing sample", () => {
    test("Snapshot testing of AdminHome component", () => {
      const renderedComponent = renderer.create(<AdminHome />).toJSON();
      expect(renderedComponent).toMatchSnapshot();
    });
  });
  