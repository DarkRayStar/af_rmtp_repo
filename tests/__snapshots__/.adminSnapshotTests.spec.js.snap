// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Snapshot testing sample Snapshot testing of AdminHome component 1`] = `
<div
  style={
    Object {
      "marginTop": "20px",
    }
  }
>
  <center>
    <h2>
      ADMIN HOME
    </h2>
    <br />
    <br />
    <a
      href="/admin-file-upload"
      onClick={[Function]}
    >
      <button
        class="btn btn-info btn-lg"
        style={
          Object {
            "marginRight": "50px",
            "width": "40%",
          }
        }
        type="button"
      >
        File Upload
      </button>
    </a>
    <a
      href="/admin-file-download"
      onClick={[Function]}
    >
      <button
        class="btn btn-info btn-lg"
        style={
          Object {
            "width": "40%",
          }
        }
        type="button"
      >
        Download Files
      </button>
    </a>
    <a
      href="/admin-add-new-submission-type"
      onClick={[Function]}
    >
      <button
        class="btn btn-success btn-lg"
        style={
          Object {
            "marginRight": "50px",
            "marginTop": "30px",
            "width": "40%",
          }
        }
        type="button"
      >
        Add Submission Type
      </button>
    </a>
    <a
      href="/admin-submission-type-list"
      onClick={[Function]}
    >
      <button
        class="btn btn-success btn-lg"
        style={
          Object {
            "marginTop": "30px",
            "width": "40%",
          }
        }
        type="button"
      >
        View Submission Types
      </button>
    </a>
    <a
      href="/admin-marking-upload"
      onClick={[Function]}
    >
      <button
        class="btn btn-primary btn-lg"
        style={
          Object {
            "marginRight": "50px",
            "marginTop": "30px",
            "width": "40%",
          }
        }
        type="button"
      >
        Add Marking Scheme
      </button>
    </a>
    <a
      href="/admin-marking-download"
      onClick={[Function]}
    >
      <button
        class="btn btn-primary btn-lg"
        style={
          Object {
            "marginTop": "30px",
            "width": "40%",
          }
        }
        type="button"
      >
        View Marking Schemes
      </button>
    </a>
    <a
      href="/admin-all-students"
      onClick={[Function]}
    >
      <button
        class="btn btn-danger btn-lg"
        style={
          Object {
            "marginRight": "50px",
            "marginTop": "30px",
            "width": "40%",
          }
        }
        type="button"
      >
        View Students
      </button>
    </a>
    <a
      href="/admin-all-employees"
      onClick={[Function]}
    >
      <button
        class="btn btn-danger btn-lg"
        style={
          Object {
            "marginTop": "30px",
            "width": "40%",
          }
        }
        type="button"
      >
        View Employees
      </button>
    </a>
    <a
      href="/admin-group-list"
      onClick={[Function]}
    >
      <button
        class="btn btn-warning btn-lg"
        style={
          Object {
            "marginTop": "30px",
            "width": "85%",
          }
        }
        type="button"
      >
        Allocate Panel Members
      </button>
    </a>
  </center>
</div>
`;
